app/views/admin/merchants/index.html.erb

```diff
@@ -1,5 +1,11 @@
 <% @merchants.each do |merchant| %>
   <ul>
```

Comment:
I noticed yesterday that this is a mistake. <ul> should be outside the loop, or it will create a new list for every merchant.